_posts/2017-10-19-haskell-learning-group-2nd-meeting.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title:  "Haskell Learning Group: Beyond Maths (2nd Session)"
+date: 2017-10-23 00:03:30 +0200
+categories: haskell learn
+author_name: Sergii Paryzhskyi
+author_url : /author/sergii_paryzhskyi
+author_avatar: sergii_paryzhskyi
+show_avatar : true
+read_time : 15
+show_related_posts: false
+square_related: recommend-wolf
+feature_image: posts/2017-10-17-first-haskell-meeting/haskell-poster.jpg
+---
+
+Previous [blog post][previous-blog-post] was an introduction to our Haskel Learning Group. That is a new initiative we started at HolidayCheck recently. As the first chapter was about mathematical concepts and lambda calculus, we haven't touched or seen any Haskell code by then. For the second meeting we had to prepare chapters 2 and 3 from the book. They were all about programming and tooling that can be used to run Haskell code. 
+For us right now REPL is more than enough to play with examples and solve exercises. It seems like most of the people were switching back and forth between pdf-book and REPL to try things out. That was also an agreement of the learning group. Doesn't matter how obvious/easy/boring things are - do not skip parts and commit yourself to doing all exercises at the end of each chapter.
+
+[previous-blog-post]: http://techblog.holidaycheck.com/post/2017/10/17/first-haskell-meeting
+
+## Tooling
+At this point we don't need to know or to use many different tools, so REPL was good to start with. REPL stands for **r**ead-**e**val-**p**rint **l**oop, an interactive programming shell that evaluates expressions and returns results in the same environment. For Haskell this is [GHCi][ghci]. You get it by installing [stack][stack]. Stack also provides many other things that we will be using in the future. It can be used for installing packages (including dependencies to specific project), and also building, testing and benchmarking your project.
+For now we will be using it only for REPL and running tests later on.
+You can start REPL in your console by running `ghci`. You can begin to play there by executing simple expressions and see results. If you feel more comfortable writing code in your editor of choice, feel free to do so. You can execute modules later on by running this in ghci: `:load filename.hs`
```

Comment:
You aren't really "executing" modules - you are (as the name suggests) loading them.

Reply:
Except if its `main`, which will be executed after it's been loaded, but you are right.